src/reporters/apiary-reporter.coffee

```diff
@@ -132,10 +132,16 @@ class ApiaryReporter
       data = @_transformTestToReporter test
       data.result = 'error'
 
+      data['resultData']['result']['general'] ?= []
+
       if CONNECTION_ERRORS.indexOf(error.code) > -1
-        data.resultData.errors.push "Error connecting to server under test!"
+        data['resultData']['result']['general'].push {
```

Comment:
Maybe a comment about "using similar interface as Gavel to keep better interoperability" would be nice.